### DISCUSSION TITLE
Download original file name in admin

### DISCUSSION THREAD

**Discussion by andre-dasilva** [django-cms/django-filer#1483] [General]
Whats the reason behind downloading the original file name in the admin?

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/e44e836f-d74b-47af-91a5-76d2961cce78">

Doesn't it make more sense to download the current url (so just `download="{{ original.url }}"`) ? So that a user can change the file name in the admin and download it.

Otherwise a user needs to reupload the same file again to get a different original name (E.g when a user want to rename the file). Which is just more confusing and more work for an end user.

**Comment by fsbraun** [comment]
@andre-dasilva I'm not sure if I understand your point. 

The `download` attribute specifies the name of the file that is created upon download. The file is downloaded from the `href` location, which is its URL.

I would expect the user to be able to name the file and then still download it. 

If the `download` attribute was a URL containing slashes, the browser would need to convert them into something the local file system can work with, probably leading to a very cryptic file name.

Maybe you can clarify?

**Comment by andre-dasilva** [comment]
@fsbraun thanks for your reply. my tests were wrong. its all working as expected 😄. i will close this discussion
